starlette/schemas.py

```diff
@@ -65,14 +77,40 @@ def get_endpoints(
                     endpoints_info.append(
                         EndpointInfo(route.path, method.lower(), func)
                     )
+
         return endpoints_info
 
-    def parse_docstring(self, func_or_method: typing.Callable) -> dict:
+    def _extract_yaml(self, docstring: str) -> str:
+        """
+        We support having regular docstrings before the schema definition.
+
+        Here we return just the schema part from a docstring.
+        """
+        yaml_docs = docstring.split("---")
```

Comment:
Can we just do `docstring.split("---")[-1]` ?

Reply:
Very True, changed.